src/utils/file_utils.py

```diff
@@ -0,0 +1,14 @@
+# Python Imports
+import yaml
+from pathlib import Path
+
+# Project Imports
+
+
+def read_yaml_file(file_path: str):
+    path = Path(file_path)
+
+    with open(path, 'r') as file:
+        data = yaml.safe_load(file)
+
+    return data
```

Comment:
Afaik, `Path` takes care of handling path separators and other OS-specific details.

Reply:
Sorry, could've phrased that better. What I wanted to say is, why not directly ask for a `Path` object as the parameter?

Reply:
Because this will probably be called either by an argument given by a config file or CLI, so we can use this function directly with that. Don't really see a benefit of extracting that line to outside.